testsuite/classlibrary/TestNodeProxy_Server.sc

```diff
@@ -54,6 +54,22 @@ TestNodeProxy_Server : UnitTest {
 		this.assertEquals(build, false, "SynthDef should not be rebuilt when calling send");
 	}
 
+	test_synthDef_isReleased_afterFree {
+		var numBefore, numAfter;
+		server.sendStatusMsg;
+		server.sync;
+		numBefore = server.statusWatcher.numSynthDefs;
```

Comment:
since these few lines of code consumed quite a bit of time in discussion, could you please add a brief comment indicating why `server.sendStatusMsg` is here so the next person to read it understands it more readily?

Reply:
sure!